my-app/src/components/episodes/EpisodeList.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from "react"
+import EpisodeCard from "./EpisodeCard"
+import axios from "axios"
+
+const EpisodeList = () => {
+  const [EpisodeList, setEpisodeList] = useState([])
```

Comment:
this does not look correct.
Avoid naming the state with the Component name. it may lead to unpredictable sideeffects.
so you could name it:

```suggestion
const [episodes, setEpisodes] = useState([])
```